Core/fft_mt_r2iq.cpp

```diff
@@ -235,11 +235,11 @@ void fft_mt_r2iq::Init(float gain, int16_t **buffers, float** obuffers)
 			// Bw *= 0.8f;  // easily visualize Kaiser filter's response
 			KaiserWindow(halfFft / 4 + 1, Astop, relPass * Bw / 128.0f, relStop * Bw / 128.0f, pht);
 
-			float gainadj = gain  / sqrtf(2.0f) * 2048.0f / (float)FFTN_R_ADC; // reference is FFTN_R_ADC == 2048
+			float gainadj = gain * 2048.0f / (float)FFTN_R_ADC; // reference is FFTN_R_ADC == 2048
```

Comment:
why？don't know what sqrt2 is for.

Reply:
I placed pht[t] both in  pfilterht[][0] and  pfilterht[][1]. This gives a real signal with sqrt2 gain, that is not present if we use
pht[t]  in  pfilterht[][0] and   pfilterht[][1] = 0. 
The index position of pfilterht[] was wrong too it is now [halfFft-1-t].
There was many errors compensating each other :-))